### PR TITLE
Archived walkthrough on embedding Office type information

### PR DESCRIPTION
**Pull request by rpetrusha**

### CODE REVIEW

File: .openpublishing.redirection.json [hunk 1]
```diff
@@ -2276,6 +2280,10 @@
             "source_path": "docs/visual-basic/programming-guide/concepts/assemblies-gac/index.md",
             "redirect_url": "/dotnet/standard/assembly"
         },
+        {
+            "source_path": "docs/visual-basic/programming-guide/concepts/assemblies-gac/walkthrough-embedding-type-information-from-microsoft-office-assemblies-in-vs.md",
+            "redirect_url": "https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120)"
```
Comment (mairaw) [inline .openpublishing.redirection.json]:
```suggestion
            "redirect_url": "https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120)"
```

File: .openpublishing.redirection.json [hunk 2]
```diff
@@ -426,15 +426,19 @@
...
-            "redirect_url": "/dotnet/csharp/async/"
-        },
         {
             "source_path": "docs/csharp/programming-guide/concepts/assemblies-gac/index.md",
             "redirect_url": "/dotnet/standard/assembly",
             "redirect_document_id":true
         },
+        {
+            "source_path": "docs/csharp/programming-guide/concepts/assemblies-gac/walkthrough-embedding-type-information-from-microsoft-office-assemblies.md",
+            "redirect_url": "https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120)"
```
Comment (BillWagner) [inline .openpublishing.redirection.json]:
```suggestion
            "redirect_url": "https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120)"
```

File: docs/csharp/programming-guide/interop/walkthrough-office-programming.md [hunk 3]
```diff
@@ -199,7 +199,7 @@ You must have Microsoft Office Excel and Microsoft Office Word installed on your
 - [Lambda Expressions (Visual Basic)](../../../visual-basic/programming-guide/language-features/procedures/lambda-expressions.md)
 - [Lambda Expressions (C#)](../../../csharp/programming-guide/statements-expressions-operators/lambda-expressions.md)
 - [How to: Use Indexed Properties in COM Interop Programming](../../../csharp/programming-guide/interop/how-to-use-indexed-properties-in-com-interop-rogramming.md)
-- [Walkthrough: Embedding Type Information from Microsoft Office Assemblies in Visual Studio (C#)](../../../csharp/programming-guide/concepts/assemblies-gac/walkthrough-embedding-type-information-from-microsoft-office-assemblies.md)
+- [Walkthrough: Embedding Type Information from Microsoft Office Assemblies in Visual Studio](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120))
```
Comment (BillWagner) [inline docs/csharp/programming-guide/interop/walkthrough-office-programming.md]:
```suggestion
- [Walkthrough: Embedding Type Information from Microsoft Office Assemblies in Visual Studio](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2013/ee317478(v%3dvs.120))
```

**Review by BillWagner — APPROVED** [review]
I agree with archiving this content @rpetrusha 

I found a few `en-us` segments in the redirected URLs. If they should be removed, I made suggestions (as did Maira for one of them.)

**Pull request merged**